### DISCUSSION TITLE
Identify processor architecture

### DISCUSSION THREAD

**Discussion by karlbrown-va** [twpayne/chezmoi#3785] [Q&A] [answered]
It would be useful to have a data field for the processor architecture, this is useful if there are different paths (for example Homebrew) that install to different places whether intel or Apple silicon. For now I'm going to make this a custom data element but wondering if this has been considered in the past?

**Comment by bradenhilton** [comment, marked as answer]
Does [`{{ .chezmoi.arch }}`](https://www.chezmoi.io/reference/templates/variables/) not achieve this already?

I also think https://github.com/twpayne/chezmoi/pull/3378#issuecomment-1835741449 has some relevance here.

**Reply by karlbrown-va** [reply]
Thanks! Sorry I had missed that param. That will work for my case.
